src/afl-fuzz-queue.c

```diff
@@ -268,14 +269,17 @@ void update_bitmap_score(afl_state_t *afl, struct queue_entry *q) {
 void cull_queue(afl_state_t *afl) {
 
   struct queue_entry *q;
-  u8                  temp_v[MAP_SIZE >> 3];
+  u32                 len = (afl->fsrv.map_size >> 3);
   u32                 i;
+  u8 *                temp_v;
 
   if (afl->dumb_mode || !afl->score_changed) return;
 
+  temp_v = ck_alloc(afl->fsrv.map_size >> 3);
```

Comment:
I suggest (re)using `ck_maybe_grow` on a buf in `afl_state_t`  here, instead of an allocation each time.

Reply:
I have no clue what that is :) too much happening in our code :) can you fix it?